styles/sideBar.css

```diff
@@ -4,13 +4,37 @@
 
 .sidebar{
     font-family: 'Supermercado One', cursive;
-    flex: 0 0 240px;
+    flex: 0 0 20%;
     margin-right: 1rem;
-    background-color: #fff;
+    /*background-color:skyblue;
+    /*width: fit-content;
+    height:max-content;  */
```

Comment:
keep the code clean. delete commented out codes

Reply:
ok i will do so